src/IJulia.jl

```diff
@@ -198,7 +202,7 @@ function clear_history(indices)
 end
 
 # since a range could be huge, intersect it with 1:n first
-clear_history{T<:Integer}(r::AbstractRange{T}) =
+clear_history(r::AbstractRange{T}) where {T<:Integer} =
```

Comment:
Can just be `clear_history(r::AbstractRange{<:Integer})` since we don't use `T`.